src/Http/Controllers/CP/Forms/FormExportController.php

```diff
@@ -11,7 +11,7 @@ class FormExportController extends CpController
 {
     public function export($form, $type)
     {
-        $this->authorize('forms');
+        $this->authorize("view {$form->handle()} form submissions");
```

Comment:
Thanks for PR!  Thinking this should probably be `$this->authorize('view', $form)` to match `FormSubmissionsController@index`, as we already have a policy setup for viewing a form and it's submissions.

Reply:
I actually tried to use `$this->authorize('view', $form)` when I was working on this but found that even if a user doesn't have permission to view the form submissions, just permission to configure forms, they'd still be able to export the submissions which doesn't seem right to me.

I tried to hook do `$this->authorize('view', $form->submissions()->first())` but don't think I managed to get that to work.

Reply:
That's intentional behaviour, as `configure forms` is kind of like `super` permissions on the form level.  If you can configure forms, you should be able to view a form and it's submissions.  (If you want to see where this is happening in the code, look at the `before()` method of `FormPolicy` and `FormSubmissionPolicy`.)

Reply:
I should note that this concept applies elsewhere as well.  eg)  `configure collections` is the same super-like behaviour to collection/entry level permissions, etc.

Reply:
Okay, I'll update it to do use that then.